limited/sly fox

```diff
@@ -0,0 +1,10 @@
+**Sly Fox** | Werewolf Power
+__Basics__
+Each day the Sly Fox may choose a player and hide in their house, protecting them from any kill targeted at them but not the house they are hidden in, the wolfpack is informed at the start of day if anyone tried to kill the Sly Fox that night. Once per game, the Sly Fox may alternatively join the wolfpack as a regular wolf. 
+__Details__
+Each day the Sly Fox may ever hide in someone's house that night or choose to join the wolfpack. 
+When the Sly Fox hides in someone's house they will be immune to any attack targeting them but will be killed if the person they hide behind is killed, except if their target is attacked by a wolf then the Sly Fox will turn into a wolf. 
```

Comment:
I feel like that's a bit too powerful (turning into a wolf part)

Reply:
It could make sense if the Wolfpack is just told the name of the person the SF is hiding at

Reply:
> It could make sense if the Wolfpack is just told the name of the person the SF is hiding at

I tried not to as it discourages the pack from attacking that player if the fox is hidden there but maybe the role is not supposed to hide from wolf kill but instead hid from solo killers or town killers

Reply:
> > It could make sense if the Wolfpack is just told the name of the person the SF is hiding at
> 
> I tried not to as it discourages the pack from attacking that player if the fox is hidden there but maybe the role is not supposed to hide from wolf kill but instead hid from solo killers or town killers

I mean... That's the point.

Reply:
Ok I'll change it